static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,853 @@
+
+.hosted-page {
+    font-family: Helvetica, Arial sans-serif;
+}
+
+.hosted-page--bright.hosted-gallery-page .hosted-tone-btn,
+.hosted-page--bright .hosted__banner .hosted-tone-btn,
+.hosted-page--bright .hosted-tone-btn:focus {
+    color: #ffffff;
+}
+
+.hosted-page--bright.hosted-gallery-page .hosted-tone-btn svg,
+.hosted-page--bright .hosted__banner .hosted-tone-btn svg,
+.hosted-page--bright .hosted-tone-btn:focus svg {
+    fill: #ffffff;
+}
+
+.hosted-page--bright .hostedbadge.hosted-tone-bg .hostedbadge__info {
+    color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header {
+    color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header .site-message__close-btn {
+    border-color: #ffffff;
+}
+
+.hosted-page--bright .survey-text__header .inline-cross {
+    fill: #ffffff;
+}
+
+.hosted__glogo {
+    display: inline-block;
+    padding: 4px 10px;
+    background-color: #005689;
+    text-align: right;
+}
+
+.hosted__glogo svg {
+    fill: #ffffff;
+    width: 115px;
+    height: 40px;
+}
+
+.hosted__glogo .inline-logo {
+    display: inline-block;
+    margin-top: -5px;
+}
+
+.hosted__glogotext {
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: .5px;
+    line-height: 16px;
+    margin: 0;
+    text-align: left;
+    color: rgba(255, 255, 255, .5);
+}
+
+.hostedbadge {
+    position: absolute;
+}
+
+.hostedbadge__logo.hostedbadge__logo {
+    position: relative;
+}
+
+.hosted__heading {
+    margin: 0;
+    font-size: 22px;
+    line-height: 24px;
+}
+
+.hosted__banner {
+    height: 200px;
+    position: relative;
+    background-size: cover;
+    background-position: left;
+}
+
+.hosted__banner:after {
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    left: 0;
+    top: 0;
+    z-index: 0;
+    background: linear-gradient(transparent 100px, rgba(0, 0, 0, .5));
+}
+
+.hosted__cta-wrapper {
+    padding-top: 140px;
+    position: relative;
+    z-index: 1;
+    margin-left: 20px;
```

Comment:
Since we have sass now, you can use variables defined in the https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/static/src/stylesheets/pasteup/grid-system/_src.scss file.